package.json

```diff
@@ -26,6 +27,6 @@
   "dependencies": {
     "babel-preset-react": "^6.24.1",
     "react": "^15.6.1",
-    "react-dom": "^15.6.1"
+    "react-dom": "^15.6.1",
```

Comment:
I think this is technically a JSON syntax error

Reply:
Fixed.